src/cascadia/TerminalSettingsEditor/Profiles.cpp

```diff
@@ -406,7 +464,7 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
 
     void Profiles::CursorShape_Changed(IInspectable const& /*sender*/, RoutedEventArgs const& /*e*/)
     {
-        _PropertyChangedHandlers(*this, Windows::UI::Xaml::Data::PropertyChangedEventArgs{ L"IsVintageCursor" });
+        _PropertyChangedHandlers(*this, PropertyChangedEventArgs{ L"IsVintageCursor" });
     }
```

Comment:
ideally this would be hooked up to the CursorShape changing, _NOT_ the checkbox/optionbutton changing (!)

Reply:
Actually... I'm glad you mentioned this. Now that we're subscribed to the cursor shape in the view model, we completely don't need this function. Removed.